### DISCUSSION TITLE
Timeline report

### DISCUSSION THREAD

**Discussion by Myxomopka666** [ariacom/Seal-Report#113] [General]
Greetings, 

I have events with start&end timestamps. Is it possible to visualize them in a timeline with description in each event in graphic instead like a table that will be harder to understand for the end users?

Regards.

**Comment by ariacom** [comment]
Can you check the sample: 05-Charts Gallery - Advanced
There are charts have date time for X Axis.

**Reply by Myxomopka666** [reply]
Greetings,

Let me give you an exapmple with a picture what kind of visualization I`m trying to acheave:
![image](https://github.com/user-attachments/assets/02b582ff-7aec-44ac-bab6-7a84e3e9bc54)

For each task I have start and end timestamps.
Is this possible?

Regards

**Comment by ariacom** [comment]
To achieve, you can not use charts, I think you can try to see your timeline as an HTML table result with colored cells (depending on the result).
It will require to have a result set containing all the time slots per task (then you can pivot the result in a cross tab), then you can use the Cell script (or a Task) to color/change your cell color using CSS.
Good luck. You can post a small sample for the community.
